app.rb

```diff
@@ -1,33 +1,57 @@
 require "sinatra/base"
+require "sinatra/reloader"
 
-require_relative "lib/warehouse"
-require_relative "lib/basket"
+Dir["./lib/**/*.rb"].each{|file| require file}
 
-$basket = Basket.new
-$warehouse = Warehouse.new
+module Store
 
-class App < Sinatra::Base
+  class App < Sinatra::Base
+    whouse = CreateWarehouse.new.call
+    basket = CreateBasket.new.call
 
-  get "/" do
-    erb :index
-  end
+    get "/" do
+      @wh_products = FetchProductsFromWarehouse.new.call(whouse.id)
```

Comment:
@wafcio Ehh, alright... It supposed to be a shortcut of "warehouse", I was too lazy to write it all the time...
I guess it will be the same thing with wh_products, wh_id and bk_id. Will rename it asap.